src/View/Helper/SchemaHelper.php

```diff
@@ -299,16 +299,4 @@ public function sortable(string $field): bool
 
         return !in_array($type, ['array', 'object']);
     }
-
-    /**
-     * Return unique right types from schema "relationsSchema".
-     *
-     * @return array
-     */
-    public function rightTypes(): array
```

Comment:
As the other comment, I would mark this method as deprecated without removing it

Reply:
method restored as deprecated